bin/run_docker.py

```diff
@@ -51,7 +51,10 @@ def get_submission_image(syn: synapseclient.Synapse, submission_id: str) -> str:
     docker_repository = submission.get("dockerRepositoryName", None)
     docker_digest = submission.get("dockerDigest", None)
     if not docker_digest or not docker_repository:
-        raise ValueError(f"Submission {submission_id} has no associated Docker image.")
+
+        input_error = f"Submission {submission_id} has no associated Docker image"
```

Comment:
Should we ask @vpchung for feedback on the error message?

Reply:
Thanks! I was going to suggest we return something a little more explicit, maybe something like:

```
Submission {SUB_ID} should be a Docker image, not {SUB_ENTITY_TYPE}.
```

@gaiaandreoletti @rrchai what do you think?

Reply:
Putting this in for now!